dist/c-jwt.spec

```diff
@@ -0,0 +1,62 @@
+Summary:            JSON Web Token (C implementation)
+Name:               c-jwt
+Version:            1.8.0
+Release:            0%{?dist}
+License:            LGPL v3
+Group:              Development/Tools
+Source:             %{name}-%{version}.tar.bz2
+URL:                https://github.com/benmcollins/libjwt
+BuildRequires:      make libtool automake
+BuildRequires:      gcc doxygen
+BuildRequires:      openssl-devel jansson-devel check-devel
+Requires:           openssl jansson check
```

Comment:
The package does not require check. Also, the built-in dependency checking of rpmbuild will add the proper library requires (i.e. openssl and jansson) so they doesn't need to be explicit.

Reply:
There is a check over check in configure.ac:

PKG_CHECK_MODULES([CHECK], [check >= 0.9.4], [true], [true])
so what is true ?

Reply:
Check is a `BuildRequires` not a `Requires`. The former is for building the package, the latter is for installing the package. The `check` library is only used to build and run the test suite, which isn't needed at install. Just remove the `Requires` line here.